lib/StaticNarrative/StaticNarrativeImpl.py

```diff
@@ -1,29 +1,23 @@
 # -*- coding: utf-8 -*-
-#BEGIN_HEADER
-import logging
-import os
-from StaticNarrative.exporter.exporter import NarrativeExporter
-from StaticNarrative.uploader.uploader import upload_static_narrative
-from StaticNarrative.narrative_ref import NarrativeRef
+# BEGIN_HEADER
```

Comment:
Ran black on the files without thinking... hence the templated text has also been formatted.

Reply:
Yikes. Is that going to break things? I seem to recall the template builder / rebuilder breaking when it sees a space after the hash.

Or is there an SDK version now that fixes that? I also remember some work being done around that.

Reply:
The SDK now accepts proper python formatting so it's not a problem. :D